docs/maya/simple_setup.md

```diff
@@ -468,3 +468,34 @@ To have a smooth transition from the simulated mesh to the animated mesh, smooth
 </figure>
 
 With this basic paint setup the AdnSkinMerge deformer will now show the results of skin simulation transferred to the final mesh.
+
+## AdnGlue
+
+To make the simulated muscles behave more compact and avoid large gaps between them, an AdnGlue node can be used. To create a basic scenario using the AdnGlue node, start with a scene with the following elements:
+
+  - A list of simulated muscles to be glued together.
+
+The AdnGlue node will take all the simulated muscles provided as inputs and generate one combined output mesh with glue constraints applied.
+
+### Create Node
+
+To create the AdnGlue node, select the simulated muscles and press the ![AdnGlue](images/adn_glue.png){style="width:4%"} shelf button or go to AdonisFX Menu > *Solvers* > *Glue*. This will connect the selected muscles to the inputs plug of the AdnGlue node and create a combined output mesh as the result of the simulation.
+
+After the node creation, input muscles can be added or removed from the existing AdnGlue by pressing AdonisFX > Glue > *Add Inputs* or AdonisFX > Glue > *Remove Inputs* respectively.
+
+### Paint Weights
+
+> [!NOTE]
+> AdnGlue requires the use of the Maya Paint tool (not the AdonisFX paint tool) for the painted weights setup.
+
+Once the AdnGlue node is properly created, you can use Maya Paint Tool to paint its weights and correctly set up the node properties. With the default paint settings provided when creating a new AdnGlue, only distance constraints will be applied, so the output mesh will look the same as the input meshes.
```

Comment:
As I mentioned before, at this point, max glue distance should be configured with a value greater than 0.0. Then we have to adjust this sentence:
```suggestion
Once the AdnGlue node is properly created, you can use Maya Paint Tool to paint its weights and correctly set up the node properties. With the max glue distance previously adjusted, the default values of the paintable maps already allow the node to compute the glue constraints.
```